src/index.js

```diff
@@ -0,0 +1,36 @@
+import './style.css';
+
+const listItems = document.querySelector('.to-do-list');
+const toDoList = [
+  {
+    index: 0,
+    completed: 'true',
+    description: 'wash the dishes',
+  },
+  {
+    index: 1,
+    completed: 'true',
+    description: 'cook food',
+  },
+  {
+    index: 2,
+    completed: 'true',
+    description: 'complete project',
+  },
+];
```

Comment:
- [ ] Good work so far, on load our list is great rendered to the DOM :clap:  But from the project requirement you need to render them in the order of their index. When I changed the index value the order doesn't change :cry:  Please fix it in order to make your project work as expected :smile:

Reply:
Fixed.